nautobot/dcim/tables/devices.py

```diff
@@ -267,14 +330,15 @@ class PathEndpointTable(CableTerminationTable):
     )
 
 
-class ConsolePortTable(DeviceComponentTable, PathEndpointTable):
+class ConsolePortTable(ModularDeviceComponentTable, PathEndpointTable):
     tags = TagColumn(url_name="dcim:consoleport_list")
 
-    class Meta(DeviceComponentTable.Meta):
+    class Meta(ModularDeviceComponentTable.Meta):
         model = ConsolePort
         fields = (
             "pk",
             "device",
+            "module",
             "name",
             "label",
             "type",
```

Comment:
Similarly here, could we define the common fields in `ModularDeviceComponentTable.Meta` and just extend/modify them inline as needed?